emcc.py

```diff
@@ -3443,11 +3444,13 @@ def phase_binaryen(target, options, wasm_target):
                           args=passes,
                           debug=intermediate_debug_info)
     building.save_intermediate(wasm_target, 'byn.wasm')
-  elif strip_debug or strip_producers:
+  elif strip_debug or strip_producers or strip_name:
```

Comment:
Do we not strip name when we run `wasm-opt` (in the `if` case above), even if `strip_name` is True?